src/desktop/apps/order/client.js

```diff
@@ -58,6 +58,7 @@ buildClientApp({
   context: {
     user: sd.CURRENT_USER,
     mediator,
+    isEigen: sd.EIGEN,
```

Comment:
also noticed that this was only being injected on the server, not on the client 😬